javascript/public/manifest.json

```diff
@@ -1,25 +1 @@
-{
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
-}
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

Comment:
I wonder if, instead of "name" we might put `"Mapache Search"`

And instead of `"short_name"` we might put `"mapache"`

Reply:
You may also want to reformat the JSON.  

Your editor might have a feature to do this automatically.  If it's VSCode, you can install a formatter for JSON files as an Extension.

Or you could do it manually.  Compare how the old manifest.json was formatted.

Reply:
Ooops.. my bad, I should have "UCSB CS LAs" for name, and "CS LAs" for short name.